internal/file/watch/watch.go

```diff
@@ -145,7 +152,10 @@ func (w *watch) Start(ctx context.Context) (<-chan error, error) {
 			case err, ok = <-w.w.Errors:
 			}
 			if !ok {
-				w, err = w.init()
+				iw, err := w.init()
+				if err == nil {
+					w = iw
+				}
```

Comment:
same. Please tell me the reason.

Reply:
sure.
if w.init returns error, w become nil.
and when roop continues, nil pointer exception will occur.